File: plugin/src/test/fixtures/junitPlatform-plugin-kotlin-mpp/src/jvmTest/java/com/example/AddTest.java
package com.example;

import static com.example.Adder.add;
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public final class AddTest {
	@Test
	public void simple() {
		assertEquals(5, add(3, 2));
	}
}

File: plugin/src/test/fixtures/plugin-kotlin-mpp-with-burst/src/commonTest/kotlin/com/example/AddTest.kt
package com.example

import app.cash.burst.Burst
import kotlin.test.Test
import kotlin.test.assertEquals

@Burst
class AddTest(
	private val strategy: Strategy,
) {
	@Test fun simple() {
		assertEquals(5, strategy.add(3, 2))
	}

	enum class Strategy {
		Language {
			override fun add(a: Int, b: Int): Int = a + b
		},
		Library {
			override fun add(a: Int, b: Int): Int = com.example.add(a, b)
		},
		;

		abstract fun add(a: Int, b: Int): Int
	}
}
